ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplAck.java

```diff
@@ -23,7 +23,9 @@
 public enum ReplAck {
     DUMP_ACKNOWLEDGEMENT("_finished_dump"),
     EVENTS_DUMP("_events_dump"),
-    LOAD_ACKNOWLEDGEMENT("_finished_load");
+    LOAD_ACKNOWLEDGEMENT("_finished_load"),
+    RANGER_DUMP_ACKNOWLEDGEMENT("_finished_ranger_dump"),
```

Comment:
We can use same DUMP_ACKNOWLEDGEMENT LOAD_ACKNOWLEDGEMENT

Reply:
As discussed we will remove the Acks